No such file `unaligned.h` in version 0.13.0

Hello, this is a homebrew maintainer and we get some error during test.

```
  ==> /usr/bin/g++-12 -std=c++17 test.cpp -o test -L/home/linuxbrew/.linuxbrew/Cellar/s2geometry/0.13.0/lib -ls2 -L/home/linuxbrew/.linuxbrew/opt/abseil/lib -labsl_log_internal_message
  In file included from /home/linuxbrew/.linuxbrew/opt/s2geometry/include/s2/util/coding/coder.h:39,
                   from /home/linuxbrew/.linuxbrew/opt/s2geometry/include/s2/s1angle.h:28,
                   from /home/linuxbrew/.linuxbrew/opt/s2geometry/include/s2/s2cell_id.h:41,
                   from /home/linuxbrew/.linuxbrew/opt/s2geometry/include/s2/mutable_s2shape_index.h:39,
                   from /home/linuxbrew/.linuxbrew/opt/s2geometry/include/s2/s2loop.h:34,
                   from test.cpp:1:
  /home/linuxbrew/.linuxbrew/opt/s2geometry/include/s2/util/endian/endian.h:23:10: fatal error: s2/util/gtl/unaligned.h: No such file or directory
     23 | #include "s2/util/gtl/unaligned.h"
        |          ^~~~~~~~~~~~~~~~~~~~~~~~~
  compilation terminated.
  Error: s2geometry: failed
```

Can you take a look?
Thanks,

- https://github.com/Homebrew/homebrew-core/pull/256573

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -451,6 +451,7 @@ if(S2_ENABLE_INSTALL)
                 src/s2/util/gtl/densehashtable.h
                 src/s2/util/gtl/hashtable_common.h
                 src/s2/util/gtl/requires.h
+                src/s2/util/gtl/unaligned.h
           DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/s2/util/gtl")
   install(FILES src/s2/util/hash/mix.h
           DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/s2/util/hash")

